src/chigame/api/tests/factories.py

```diff
@@ -1,5 +1,6 @@
 import random
```

Comment:
Do we use the random import anywhere? Else, it is not really necessary!

Reply:
Taken care of!